### PR TITLE
Use --max_file_size=-1 to avoid splitting export files

### LINKED ISSUE #8: --max_file_size=0 should not split files
Since it makes no sense, it should simply be the way to disable file splitting (better than using --max_file_size=99999)

### PR DESCRIPTION
**Pull request by drzraf**
Fixes #8

### CODE REVIEW

**Comment by miya0001** [comment]
Can you add tests too?

**Comment by drzraf** [comment]
updated (not sure about behat's test, couldn't run the tests yet).
`-1` is the documented value, but 0 has the same (undocumented) effect since it seems more logical to me (and may be a sane behaviour for possible future reuse)

File: features/export.feature [hunk 1]
```diff
@@ -479,3 +479,21 @@ Feature: Export content.
       """
       0
       """
+
+  Scenario: Export spliting the dump
```
Comment (danielbachhuber) [inline features/export.feature]: This will need `Given a WP install` to set up the WordPress install.

File: src/Export_Command.php [hunk 2]
```diff
@@ -382,7 +383,7 @@ private function check_max_file_size( $size ) {
 			return false;
 		}
 
-		$this->max_file_size = $size;
+		$this->max_file_size = $size <= 0 ? WP_EXPORT_NO_SPLIT : $size;
```
Comment (danielbachhuber) [inline src/Export_Command.php]: Let's make `-1` the only supported argument to prevent unexpected, undocumented behavior.

Comment (drzraf) [reply src/Export_Command.php]: what to do with --max-file-size=0 ?
Isn't a zero or negative value explicit enough to keep from spliting dumps?

**Comment by drzraf** [comment]
new commits (don't know if you received notification about this)

**Comment by danielbachhuber** [comment]
@drzraf Is this completed to your satisfaction?

**Comment by gitlost** [comment]
A problem is that `WP_EXPORT_NO_SPLIT` isn't getting defined if `KB_IN_BYTES` is defined, which is the case for modern WPs but not for older ones such as 3.7.11. This isn't visible currently on Travis, only locally in `error_log`. Another problem is that `max_file_size` is getting multiplied by `MB_IN_BYTES`, with no check for `WP_EXPORT_NO_SPLIT`.

A problem with the tests is that they need a database > 15MB to check the no-split versus the default.

I made changes (eventually) to address these in https://github.com/gitlost/export-command/pull/2, which I can push if desired.

**Comment by danielbachhuber** [comment]
> I made changes (eventually) to address these in edit gitlost#2 gitlost#3, which I can push if desired.

Please do!

**Comment by gitlost** [comment]
> which I can push if desired.

Okay, I don't know how to do this on this branch, even after reading https://help.github.com/articles/committing-changes-to-a-pull-request-branch-created-from-a-fork/, so I'll open a new PR.

**Pull request merged**